src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/ValueDistFunction.java

```diff
@@ -47,8 +47,7 @@ public void addInterestLocation(LatLng interest) throws Exception {
 			maxDuration = currentdistance;
 		double utility = 1 - setUtility(currentdistance);
 		interestlocation.put(interest, utility);
-		LOGGER.info("The interest location (" + interest + ") with the utility " + utility
-				+ " has been had with success in the Map.");
+		LOGGER.info("The interest location (", interest, ") with the utility ", utility, " has been added with success in the Map.");
```

Comment:
Attention cela va juste retourner : 
`17:34:49.756 [main] INFO  i.g.o.y.a.apartment.ApartmentFactory - The interest location (`
cf [Documentation Slf4J Binder](http://www.slf4j.org/manual.html#typical_usage)

Reply:
Fixed on commit 5658102